test/tests/props.js

```diff
@@ -813,6 +813,73 @@ describe("zoid props cases", () => {
     });
   });
 
+  it("should pass a trustedDomain prop and have it populate on the child", () => {
```

Comment:
we should add a test where the domains don't match 

`url: "mock://www.child.com/base/test/windows/child/index.htm",`
`trustedDomains: ["mock://www.different-domain.com"],`